src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/dto/InnvilgelseTilsynBarnDto.kt

```diff
@@ -9,3 +10,7 @@ data class InnvilgelseTilsynBarnResponse(
 
 data object InnvilgelseTilsynBarnRequest :
     VedtakTilsynBarnRequest, VedtakTilsynBarnDto(TypeVedtak.INNVILGELSE)
+
+data class InnvilgelseTilsynBarnRequestV2(
+    val vedtaksperioder: List<VedtaksperiodeDto>,
```

Comment:
Kunde eventuellt bare gjenbrukt den opprinnelige requesten og lagt til en nullable vedtaksperioder

Husk at vi må returnere vedtaksperioder i responsen og
Og vil ha behov for å vise "forrige vedtaksperioder" på lik måte som i læremidler i frontend 😁 
Jeg la til det til favro-oppgaven

Reply:
Ser "forrige vedtaksperioder" er håndtert i frontend 👍 
https://github.com/navikt/tilleggsstonader-sak-frontend/pull/698/files#diff-a5a301468d5c9bb80e5187adf007ecb1dbde4a3d9819124c819d3e04e0106f16R32